bin/setup

```diff
@@ -20,7 +20,8 @@ class Runner
       setup_development_procfile
       step "Installing dependencies...", "bundle install"
       step "Installing packages...", "npm install"
-      step "Configuring databases...", "hanami db prepare"
+      step "Configuring databases...", "bundle exec hanami db prepare"
+      step "Compiling assets...", "bundle exec hanami assets compile"
```

Comment:
:axe: It's safe to delete this line because the `Procfile` ensures the assets are properly compiled for production use. The other reason you don't want to compile assets during setup is because they only build for production which is not what you want for a development environment setup.